### ISSUE TITLE
Expected value of food_product_type

### ISSUE THREAD

**Issue by ramonawalls** [microbiomedata/mixs-6-2-release-candidate#165]
In reviewing mixs_table_stage modifications, I see that your suggestion for the expected value of food_product_type is FOODON:00001002 or FOODON:03309997. Does this mean that it should be FOODON:00001002 or FOODON:03309997 or one of their subclasses? If so, it needs to be phrased more clearly.

**Comment by turbomam** [comment]
I agree, but this Expected value comes directly from the MIxS excel sheets.

LinkML allows the range for a slot like `food_product_type` to be set to an enumeration that has a `reachable_from` relationship with a `ReachabilityQuery`. If we use that, then the LinkML validator can check if the values provided in  `food_product_type` (etc.) really come from subclasses of either FOODON:00001002 or FOODON:03309997.

BUT, the acceptable values will just be the ontology ids, not the "{term_label} [{term_id}]" pattern.

See also
- #dyn

**Comment by ramonawalls** [comment]
Okay, then we can leave it as is for now, but we should address this as a
more general issue some day.
